careers/technical-engineer.md

```diff
@@ -1,48 +1,78 @@
+**Engineering Perspective:** Staff-plus engineers are the folks who will often get unexpectedly pulled into the room where this sort of decision is happening. This gives them the opportunity to inject the engineering context and perspective into a decision while it's still possible to change the outcome. These brief moments of input on critical decisions are unduly impactful and will allow you to inject an engineering perspective where it would otherwise be missed.
 
-**Technical Scope:** A Staff Engineer is still acting in a very hands-on role, and as such, they are a prolific contributor to both core projects at Packback and to experimental work. When presented with a complex problem, process or existing system they are able to reduce the complexity in order to get more done with less work. They provide guidance, direction and help colleagues build and produce better outcomes.
+Concrete examples could include:
 
-**Communication:** The Staff Engineer has strong abilities to influence without requiring reporting authority. They facilitate cross-team work and are influential beyond their individual group. They are capable of driving groups of disparate interests to decisions, and clearly communicating and seeing those decisions through to impact. The staff engineer is a very strong communicator and can simplify technical concepts in a way to explain them to non-technical partners. The staff engineer is capable of setting short to medium term strategic direction for part of the technology stack, identifying areas of critical need based on future growth and developing roadmaps to attack those problems. The Staff Engineer may act as the leader of a technical team but is probably not acting as the people manager of fellow engineers.
+- Your perspective is sought out in critical decisions
+- Actively inserting yourself into these critical moments
+- Advocating for effective solutions and strategies
+- Bringing perspective from other teams or departments
 
-**Responsibilities:**
+**Mentorship:**
+
+A Staff Engineer is far more likely to change your company's long-term trajectory by growing other engineers than through personal heroics. Their effort here leads to a well-trained team with multiple experts in each domain.
+
+Effective mentorship is not a checkbox on the career ladder that can be satisfied by simply sharing advice and experience. Staff engineers build an ongoing relationship, understand the context of their mentee, and [sponsor](https://larahogan.me/blog/what-sponsorship-looks-like/) them.
+
+Concrete examples could include:
+
+- Owning onboarding and continuing education initiatives
+- No bus-factor-one concerns / building redundancy in systems and knowledge
+- Providing critical feedback on architectural design documents
+- Sponsorship
```

Comment:
Again, sponsorship is mentioned above. An example of sponsorship isn't sponsorship. Maybe we could say something like "Encouraging other engineers to take a specific online training or certification course"

Reply:
Lol. Good catch